src/ConfigManager.h

```diff
@@ -80,6 +83,10 @@ class ConfigParameter : public BaseParameter {
         }
     }
 
+    void clearData() {
+        *ptr = NULL;
```

Comment:
Should this not be the zero value of `T`?

Reply:
How would that be written? This works, but it may be jamming a square into a round ... was a bit out of my realm on this TBH

Reply:
I dont know off the top of my head, would need to look it up.

Reply:
According to https://stackoverflow.com/a/11902122, `T()` is correct.